services/QuillLMS/app/models/concerns/student.rb

```diff
@@ -137,9 +137,9 @@ def hide_extra_activity_sessions(classroom_unit_id)
     .where("users.id = ?", id)
     .where("classroom_units.id = ?", classroom_unit_id)
     .where("activity_sessions.visible = true")
-    .order("activity_sessions.is_final_score DESC, activity_sessions.percentage ASC, activity_sessions.started_at")
+    .order("activity_sessions.is_final_score DESC, activity_sessions.percentage DESC, activity_sessions.started_at")
```

Comment:
IMPORTANT NOTE: This is explicitly a change to the code, but from what I can understand from the descriptions in the specs, this is the intended behavior (and we just didn't have a case to prove that this functionality wasn't working before).  It is possible that we shouldn't try to make this change during this PR, but since I was adding the test case anyway, it felt like it should work as intended...